tests/mock_network/node_ageing.rs

```diff
@@ -296,18 +296,32 @@ fn churn_until_age_counter(
 
         match churn {
             Churn::Add => {
-                add_node_to_section(env, nodes, prefix);
+                // We are making lot of churn here and so it might happen that other section's view
+                // of this section could become so out-of-date that none of the nodes they know is
+                // still online. If we picked such node for the bootstrap node, the bootstrapping
+                // would fail because the node would keep redirecting the joining node to
+                // non-existing peers. To avoid this and to keep things simple, we make sure we
+                // bootstrap off a node from the same section.
```

Comment:
`bootstrap off a node from the same section` , don't get it, `same section` as what?

Reply:
from the same section that we are joining. I'll update the comment.